src/main/java/com/raisetech/mybatis/entity/Movie.java

```diff
@@ -0,0 +1,28 @@
+package com.raisetech.mybatis.entity;
+
+public class Movie {
+
+  private int id;
+
+  private String name;
+
+  private String published_year;
+
+  public Movie(int id, String name, String published_year) {
+    this.id = id;
+    this.name = name;
+    this.published_year = published_year;
+  }
+
+  public int getId() {
+    return id;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public String getPublished_year() {
```

Comment:
Javaのメソッド名の命名規則に従いましょう。
```suggestion
  public String getPublishedYear() {
```

Reply:
対応完了